src/routes/safe/store/actions/fetchTransactions.js

```diff
@@ -147,29 +147,54 @@ export const buildTransactionFrom = async (
     modifySettingsTx,
     customTx,
     cancellationTx,
+    creationTx: tx.creationTx,
   })
 }
 
+const addMockSafeCreationTx = (safeAddress) => [{
+  baseGas: 0,
+  confirmations: [],
+  data: null,
+  executionDate: null,
+  executor: null,
+  gasPrice: 0,
+  gasToken: '0x0000000000000000000000000000000000000000',
+  isExecuted: true,
+  nonce: null,
+  operation: 0,
+  refundReceiver: '0x0000000000000000000000000000000000000000',
+  safe: safeAddress,
+  safeTxGas: 0,
+  safeTxHash: '',
+  signatures: null,
+  submissionDate: null,
+  to: '',
+  transactionHash: null,
+  value: 0,
+  creationTx: true,
+}]
+
 export const loadSafeTransactions = async (safeAddress: string) => {
   web3 = await getWeb3()
 
-  const url = buildTxServiceUrl(safeAddress)
-  const response = await axios.get(url)
-  const transactions: TxServiceModel[] = response.data.results
+  let transactions: TxServiceModel[] = addMockSafeCreationTx(safeAddress)
+  try {
+    const url = buildTxServiceUrl(safeAddress)
+    const response = await axios.get(url)
+    if (response.data.count > 0) {
```

Comment:
`.count`? what's the type of `response.data`?

Reply:
response.data ->
```
{
 "count": <number of elements in results list>,
 "next": null,
 "previous": null,
 "results": <transaction list here>
}
```